python/lsst/daf/butler/formatters/pickleFormatter.py

```diff
@@ -77,4 +77,47 @@ def _writeFile(self, inMemoryDataset, fileDescriptor):
             The file could not be written.
         """
         with open(fileDescriptor.location.path, "wb") as fd:
-            pickle.dump(inMemoryDataset, fd, protocol=-1)
+            fd.write(self._toBytes(inMemoryDataset))
+
+    def _fromBytes(self, inMemoryDataset, pytype=None):
+        """Read the bytes object as a python object.
+
+        Parameters
+        ----------
+        pickledDataset : `bytes`
+            Bytes object to unserialize.
+        pytype : `class`, optional
+            Not used by this implementation.
+
+        Returns
+        -------
+        data : `object`
```

Comment:
Call this inMemoryDataset rather than `data` to be clear what is happening.

Reply:
Forgot (currently) but should be up in minutes.